defaults/config.js

```diff
@@ -139,6 +139,29 @@ module.exports = {
 	// This value is set to `2048` kilobytes by default.
 	prefetchMaxImageSize: 2048,
 
+	// ### `upload`
+	//
+	// Allow uploading files to the server hosting TheLounge.
+	// If enabled, The Lounge will store files in the `${THELOUNGE_HOME}/uploads` folder.
+	//
+	// Images will not expire or be removed by TheLounge, so be wary of memory and your hardware.
```

Comment:
Typo: `TheLounge` (and at L144).

Instead of `so be wary of memory and your hardware`, I would go with `which can cause issues depending on your hardware, for example in terms of disk usage.` No big deal though.